app/aboutUs.html

```diff
@@ -25,121 +25,181 @@
 <body>
    <div id="header"></div>
 
-   <div class="page-title headings white">
+   <!-- <div class="page-title headings white">
        About Us
-   </div>
-
-   <div class="centered">
-      <div class="gap"></div>
-      <div class="title">What is KOJOBS?</div>
-      <div class="gap"></div>
-
-         <div class="paragraph">
-            Kojobs is a Korean club that focuses primarily on students’ internships opportunity and employment connections. We are holding various events to provide practical assistance to students who are curious about their intended major and career path. By engaging in our club activities, students will be able to take numerous benefits, such as acquiring information regards employment and networking with professional mentors.
+   </div> -->
+
+
+  <div class="page_setting">
+      <div class="page_padding">
+         <div class="column_form">
+            <p class="about_title">
+               <br/>
+               About
+               <br/>
+               KOJOBS
+            </p>
+            <div class="gold_bar">
+            </div>
+            <p class="description">
+               Kojobs is a Korean club that focuses primarily on students’ internships opportunity and employment connections. We are holding various events to provide practical assistance to students who are curious about their intended major and career path. By engaging in our club activities, students will be able to take numerous benefits, such as acquiring information regards employment and networking with professional mentors. 
+               <br/>
+            </p>
          </div>
-      
-      <div class="gap"></div>
-      <div class="gap"></div>
-      <div class="gap"></div>
-
- 
-         <div class="divider div-transparent"></div>
+         <picture>
```

Comment:
image for aboutUs introduction is a bit big, and also seems outdated since we are not using this logo anymore. How about aligning our new logo as well as 2-3 kojobs related images horizontally?

Reply:
I agree, the logo seems little big on the web page. Another idea maybe to use @media function (refer to "https://www.w3schools.com/cssref/css3_pr_mediaquery.php") to set the logo as it is until 1000px max-width, and if its over 1000px, change into certain height and width to make it smaller and fit. Other than that, I like the design!